### PR TITLE
Radiology insights ga

### PR DESCRIPTION
**Pull request by koen-mertens**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
-
-### Features Added
-
+## 1.0.0 (2024-05-03)
```
Comment (chunyu3) [inline sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/CHANGELOG.md:3]: please update the release date.

**Review by chunyu3 — COMMENTED** [review]
Please update `README.md` in the `samples` directory https://github.com/Azure/azure-sdk-for-net/blob/60122ad2861e1a5e79b8fbe332d3a63408029ffc/sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/samples/README.md
to include all the sample code in the bottom table.
you can refer to https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/Create_Samples_Guidance.md, and this is an example https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/samples/README.md

And the sample file name should be `Sample<sample_number>_<scenario>.md` and the sample source code file should be `Sample<sample_number>_<scenario>.cs`.

File: sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/CHANGELOG.md (hunk 1 above)
Comment (rameezmicrosoft) [reply sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/CHANGELOG.md]: done

File: sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
+    - "Datetime" field on FollowupCommunication renamed into "createdAt" field
+    - Renamed createdDateTime into createdAt
+    - Renamed expirationDateTime into expiresAt
+    - Renamed lastUpdateDateTime into updatedAt
 
 ### Bugs Fixed
 
 ### Other Changes
 
-## 1.0.0-beta.1 (2024-03-01)
```
[the remark this answers is not in the document]
Comment (rameezmicrosoft) [reply sdk/healthinsights/Azure.Health.Insights.RadiologyInsights/CHANGELOG.md:13]: yes. Updated it

**Review by tomsft — APPROVED** [review]

**Pull request merged**